bin/lean-gdb.py

```diff
@@ -1,3 +1,11 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+# Released under Apache 2.0 license as described in the file LICENSE.
+#
+# Author: Sebastian Ullrich
```

Comment:
> But how do you know it is a expr_const?

Probably not at all in any automatic way. I really think calling into C++ is the simplest way to go here.